src/librbd/ImageWatcher.cc

```diff
@@ -844,7 +844,6 @@ void ImageWatcher::handle_payload(const FlattenPayload &payload,
 
 void ImageWatcher::handle_payload(const ResizePayload &payload,
 				  bufferlist *out) {
-  RWLock::RLocker l(m_image_ctx.owner_lock);
```

Comment:
@wonzhq Sorry, this fell off my radar.  I actually have a new PR that addresses all helgrind / lockdep errors in librbd here: https://github.com/ceph/ceph/pull/4528